yxi/xml/xml_generator.py

```diff
@@ -23,7 +23,7 @@ def gen_xml(yxi):
         root,
         "kernel",
         {
-            "name": yxi["toplevel"],
+            "name": "wrapper",
```

Comment:
TODO: Might need to change this from `wrapper` to `Toplevel` according to the backwards compatibility fix implemented, mentioned above.